Overview.bs

```diff
@@ -259,32 +256,36 @@ Each <a>animator instance</a> lives in an {{AnimationWorkletGlobalScope}}. The
 <a>animator instance</a> cannot be disposed arbitrarily (e.g., in the middle of running animation
 as it may contain the scripted animation state.
 
-The {{AnimationWorkletGlobalScope}} has an <dfn>animator instance list</dfn>. Anytime a new
-<a>animator instance</a> is constructed in that scope, it gets  added to the list.
+The {{AnimationWorkletGlobalScope}} has an <dfn>(animator name, root element) to instance map</dfn>.
+The map is populated when the user agent constructs a new <a>animator instance</a> in that scope.
 
-To <dfn>create a new animator instance</dfn> given a |root element|, |name|, and |workletGlobalScope|,
+To <dfn>create a new animator instance</dfn> given a |name|, |root element|, and |workletGlobalScope|,
 the user agent <em>must</em> run the following steps:
 
-    1. If an |animatorInstance| with the same |name| and |root element| exists within the <a>animator
-          instance list</a> then abort the following steps.
-
-    2. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
-          <a>animator name to animator definition map</a>.
+    1. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
+         <a>animator name to animator definition map</a>.
 
           If |definition| does not exist abort the following steps.
 
-    3. Let |animatorCtor| be the <a>class constructor</a> of |definition|.
+    2. Let |animatorInstanceMap| be |workletGlobalScope|'s <a>(animator name, root element) to
+         instance map</a>.
+
+    3. If an entry exists for |name| and |root element| within |animatorInstanceMap| then abort the
+         following steps.
+
+    4. Let |animatorCtor| be the <a>class constructor</a> of |definition|.
 
-    4. Let |animatorInstance| be the result of <a>Construct</a>(|animatorCtor|).
+    5. Let |animatorInstance| be the result of <a>Construct</a>(|animatorCtor|).
 
-          Issue: handle invalid construction.
+          If <a>Construct</a> throws an exception, set the result of looking up |name| and
+	    |root element| in |animatorInstanceMap| to <b>null</b>, and abort the following steps.
```

Comment:
What about this is repetitive? (Do you mean that you assume that the constructor will set the name and requested flag?)